src/catalog/catalog.cpp

```diff
@@ -1188,14 +1411,16 @@ void Catalog::InitializeLanguages() {
     auto &txn_manager = concurrency::TransactionManagerFactory::GetInstance();
     auto txn = txn_manager.BeginTransaction();
     // add "internal" language
-    if (!LanguageCatalog::GetInstance().InsertLanguage("internal", pool_.get(),
-                                                       txn)) {
+    if (!LanguageCatalog::GetInstance().InsertLanguage(txn,
```

Comment:
This is not your problem, but we should **not** be initializing the language table in this function.

Reply:
Ack

Reply:
Done. #1421